blocks/api/source.js

```diff
@@ -1,38 +1,66 @@
-/**
- * WordPress dependencies
- */
-import { createElement } from '@wordpress/element';
+function warnAboutDeprecatedMatcher() {
+	// eslint-disable-next-line no-console
+	console.warn(
+		'Attributes matchers are deprecated and they will be remove in future version of Gutenberg.' +
+		'Please update your attributes definition https://wordpress.org/gutenberg/handbook/reference/attributes/'
```

Comment:
Grammar: There should be a space between these two sentences, either prefixing this string or suffixing the previous line. (or separate arguments to `console.warn` which is maybe not the best behavior to rely on)